src/app/profile/settings/settings.component.less

```diff
@@ -0,0 +1,45 @@
+@import (reference) '../../../assets/stylesheets/shared/osio.less';
+
```

Comment:
Replace all hex colors with PatternFly variables. Refer to this page for color variables: http://www.patternfly.org/styles/color-palette/. Everything is preface `@color-` followed by the color's name.